.github/workflows/publish-fern-docs.yml

```diff
@@ -0,0 +1,43 @@
+name: Publish Docs
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'push' && contains(github.ref, 'refs/heads/main') && github.run_number > 1 }}
+    steps:
+      - name: Checkout aa-sdk
+        uses: actions/checkout@v4
+
+      - name: Download Alchemy Docs Repo
+        run: |
+          rm -rf docs-site && \
+          curl -L -f -H "Authorization: token ${{ secrets.DOCS_GITHUB_TOKEN }}" \
+            -o docs.tar.gz https://api.github.com/repos/alchemyplatform/docs/tarball/main && \
+          tar xzf docs.tar.gz && \
+          mv alchemyplatform-docs-* docs-site && \
+          rm docs.tar.gz
+
+      - name: Copy docs to docs-site/account-kit and cd into docs-site
+        run: |
+          mkdir -p docs-site/account-kit
+          cp -r docs/* docs-site/account-kit/
+
+      - name: Insert Account Kit docs.yml into docs-site/fern/docs.yml
+        run: |
+          cd docs-site
+          account-kit/scripts/insert-docs.sh
+
+      - name: Install Fern CLI
+        run: npm install -g fern-api
```

Comment:
if this and the previous action are going to stay long term, then we should move this into a shared composite action so it's easier to maintain

Reply:
Once the new docs is live and we don't need the GitHub token here, my thinking was to capture most of this in a script that can be repurposed for both local dev and these actions. But ya until then agreed on a composite.